flocker/node/_deploy.py

```diff
@@ -307,17 +341,22 @@ def run(self, deployer):
         return gather_deferreds(results)
 
 
-class Deployer(object):
+@implementer(IDeployer)
+class P2PNodeDeployer(object):
```

Comment:
For the initialization at least that means I have to change all constructors to be keyword-based, don't have the energy to do that at the moment.